src/pages/library/dataBrowser-utils.js

```diff
@@ -106,6 +108,17 @@ const extractTags = dataset => {
   }
 }
 
+export const filterAndNormalizeDatasets = datasets => {
+  const consortiumsToInclude = getEnabledBrand().catalogConsortiumsToInclude
+  const filteredDatasets = _.filter(dataset => consortiumsToInclude === undefined ||
+      _.intersection(consortiumsToInclude, _.map(dataCollection => dataCollection && dataCollection['dct:title'], dataset['TerraDCAT_ap:hasDataCollection'])).length > 0,
```

Comment:
If you didn't want to duplicate the code that extracts the consortium name you could filter using `project` after normalization has occurred. It's true that it will only match on the first data collection, but we only support a single consortium in the UI right now anyway. When/if we fix the UI code to handle more than one data collection per dataset we'd then change the filter code to handle that too.

Reply:
I really don't want to make the assumption. Right now, a user could put in multiple dataCollections in the API, and that would still work, and even still function on the UI (even if only the first consortium is shown.) I think we can fix the display in the UI, but I don't want to add more bugs if I can help it.